assets/js/admin/quick-edit.js

```diff
@@ -118,4 +118,13 @@ jQuery(function( $ ) {
 		}
 
 	});
+
+	$( '#wpbody' ).on( 'click', '.trash-product', function() {
+		var confirm_delete = confirm( woocommerce_admin.delete_product_message );
```

Comment:
Need to change `confirm` to `window.confirm`.

Reply:
In fact, you can change all this code to:

```js
$( '#wpbody' ).on( 'click', '.trash-product', function() {
	return window.confirm( woocommerce_admin.delete_product_message );
});
```

Reply:
implemented both the suggested changes